FreeRTOS/FreeRTOS-Plus-TCP
# Fix MISRA 64bit violations + configuration

Fix MISRA 64bit vilations + configuration

Description
-----------
Fix MISRA 64bit vilations + configuration

Test Steps
-----------
Running MISRA checker


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
